moviepy/config.py

```diff
@@ -45,10 +45,9 @@ def try_cmd(cmd):
 else:
     success, err = try_cmd([FFMPEG_BINARY])
     if not success:
-        raise IOError(err.message +
-                 "The path specified for the ffmpeg binary might be wrong")
-
-
+        raise IOError(
+            str(err) +
+            " - The path specified for the ffmpeg binary might be wrong")
```

Comment:
Wait, some have the `msg` attribute. I am not seeing any (on Windows, Python 3.6) with a `message` attribute. In any case, the `message` attribute isn't standard, and this change is still recommended.